laze-project.yml

```diff
@@ -590,6 +591,11 @@ builders:
   - name: microbit-v2
     parent: nrf52840
 
+  - name: particle-xenon
+    parent: nrf52840
+    env:
+      BOARD: particle-xenon
```

Comment:
This is not necessary, it is set in the `riot-rs` context for all builders.

Reply:
Thanks, forcepushed. (I think it sneaked in as part of fixing the missing function in the riot-ros-boards/src/lib.rs dispatch.)